src/porepy/geometry/distances.py

```diff
@@ -49,26 +57,33 @@ def segment_set(start, end):
     return d, cp
 
 
-def segment_segment_set(start, end, start_set, end_set):
+def segment_segment_set(
+    start: np.ndarray, end: np.ndarray, start_set: np.ndarray, end_set: np.ndarray
+) -> tuple[np.ndarray, np.ndarray, np.ndarray]:
     """Compute distance and closest points between a segment and a set of
     segments.
 
-    Algorithm can be found at http://geomalgorithms.com/a07-_distance.html (see
-    C++ code quite far down on that page).
+    .. note:: Algorithm can be found at http://geomalgorithms.com/a07-_distance.html
+        (see C++ code quite far down on that page).
```

Comment:
This link seems outdated to me. It only points to a site saying that the webpage is now available as a book

Reply:
Updated it.